pages/repos.js

```diff
@@ -23,7 +46,21 @@ export default function Repos({ repos }) {
 
       <Page>
         <h1 className="text-4xl mb-4 font-bold">Community Repos</h1>
-
+        <div className=" flex justify-end">
+          <Select
+            name="event-type"
+            value={router.query.sortBy || "pushed-date"}
+            label="Sort by"
+            onChange={(e) => {
+              router.push(`/repos?sortBy=${e.currentTarget.value}`);
+            }}
```

Comment:
I think this doesn't need the `{}` wrappers?

```suggestion
            onChange={(e) =>  router.push(`/repos?sortBy=${e.currentTarget.value}`)}
```

Reply:
This was marked as resolved but I don't see any changes or comment? 🤔  I will mark as unresolved